site/README

```diff
@@ -0,0 +1,25 @@
+# Building the website locally
```

Comment:
Call `README.md` since it's markdown?

Reply:
Left off the `.md` so that it would not get included in the site itself. I can add `.md` and an exclude for it though.